lab-integration/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "lab-integration",
+  "version": "1.0.0",
+  "main": "main.js",
+  "scripts": {
+    "build": "npx tsc",
+    "start": "node dist/main.js",
+    "dev": "nodemon src/main.ts",
+    "lint": "eslint .  --ext .ts",
+    "lint-fix": "eslint --fix .  --ext .ts"
+  },
+  "devDependencies": {
+    "@types/node": "^20.12.7",
+    "@typescript-eslint/eslint-plugin": "^6.21.0",
+    "eslint": "^8.57.0",
+    "eslint-config-prettier": "^9.1.0",
```

Comment:
[Prettier doc](https://prettier.io/docs/en/integrating-with-linters) says we just need `eslint-config-prettier`. Can we follow the instructions?